dev.sh

```diff
@@ -10,9 +10,59 @@
 set -e
 DIR=$(cd "$(dirname "$0")"; pwd)
 
+export LANGUAGES="en,fr"
+export LOG_LEVEL="error"
+
+if [  -n "$(uname -a | grep Ubuntu)" ]; then
+  OS="UBUNTU"
+  else
+  OS="MAC"
+fi
+
+if [ ! $OS == "UBUNTU" ]; then
+  export LOCAL_IP=host-gateway
+fi
```

Comment:
Wonder what's the purpose of this export nowadays? I'm running WSL (Ubuntu):

```
➜  opencrvs-core git:(multi-terminal-dev-workflow) uname -a
Linux DESKTOP-4TD8SBI 5.4.72-microsoft-standard-WSL2 #1 SMP Wed Oct 28 23:40:43 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

grep doesn't match Ubuntu, but this PR still runs fine on my machine. I think we should
1. ask someone with Ubuntu (@Zangetsu101 ?) to test without the conditional
2. add a comment why this is needed

Reply:
Yea, the state of this should be researched again. Originally, this was targeted for MacOS where the `172.17.0.1` address we used for referencing the host machine wasn't working. We use this mapping to enable OpenHIM to use addresses like `gateway` (just like it does in deployed environments) so that it in actuality points to the host machine's `localhost`.

This is where we use this:
https://github.com/opencrvs/opencrvs-core/blob/develop/docker-compose.demo-deps.yml#L99-L108